.github/workflows/validate.yml

```diff
@@ -0,0 +1,29 @@
+name: Validate
+on: push
+jobs:
+  validate:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - name: Get yarn cache directory path
+        id: yarn-cache-dir-path
+        run: echo "::set-output name=dir::$(yarn cache dir)"
+      - uses: actions/cache@v1
+        id: yarn-cache
+        with:
+          path: ${{ steps.yarn-cache-dir-path.outputs.dir }}
+          key: ${{ runner.os }}-yarn-${{ hashFiles('**/yarn.lock') }}
+          restore-keys: |
+            ${{ runner.os }}-yarn-
+      - name: Use Node.js
+        uses: actions/setup-node@v1
+        with:
+          node-version: '10.x'
+      - name: Install dependencies
+        run: yarn install
+      - name: Build
+        run: yarn build
+      - name: Lint
+        run: yarn lint
+      - name: Test
```

Comment:
so these steps also do storybook related validation?

Reply:
Nope, will add a step does `storybook:build` to validate it.

Reply:
![Screen Shot 2020-04-13 at 3 33 14 PM](https://user-images.githubusercontent.com/11376798/79101873-1e016700-7d9c-11ea-81a4-e3db0dd135b7.png)
is it possible to let github to display every steps instead of just one validate check?

Reply:
I don't think so.